addons/account/views/account_invoice_view.xml

```diff
@@ -91,18 +91,17 @@
                             <label for="quantity"/>
                             <div>
                                 <field name="quantity" class="oe_inline"/>
-                                <field name="uom_id" class="oe_inline" groups="uom.group_uom"/>
+                                <field name="uom_id" class="oe_inline ml-2" groups="uom.group_uom"/>
```

Comment:
I see that the quantity and uom are usually added with a "o_row" like the following example:

```
<div class="o_row">
    <field name="product_uom_qty"/>
    <field name="product_uom" groups="uom.group_uom"/>
</div>
```

And it seems to add the necessary space between the fields.
Can you change here too for consistency?